Parsers/ASimNetworkSession/Parsers/vimNetworkSessionMicrosoftLinuxSysmon.yaml

```diff
@@ -127,7 +134,7 @@ ParserQuery: |
           | where 
               (array_length(dstipaddr_has_any_prefix)==0 or has_any_ipv4_prefix(DstIpAddr, dstipaddr_has_any_prefix))
           and (array_length(srcipaddr_has_any_prefix)==0 or has_any_ipv4_prefix(SrcIpAddr, srcipaddr_has_any_prefix))
-          and (array_length(hostname_has_any)==0 or DstHostname has_any (hostname_has_any) )
+          and (array_length(hostname_has_any)==0 or DstHostname has_any (hostname_has_any)or SrcHostname has_any (hostname_has_any) )
```

Comment:
Again, I don't think you need to post filter both here and when calculating ASimMatchingHostname and ASimMatchingIpAddr . You can calculate those here.